packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -89,6 +94,13 @@ const FilterConfiguration: FC<Props> = ({
                         internalFilterRule.values.length <= 0)
                 }
                 onClick={() => {
+                    track({
+                        name: EventName.ADD_FILTER_CLICKED,
+                        properties: {
+                            mode: isEditMode ? 'edit' : 'viewer',
+                        },
+                    });
+
```

Comment:
This component is when we are editing a filter. 
Adding a filter is in packages/frontend/src/components/DashboardFilter/FilterSearch/index.tsx

Reply:
But the filter is not added until it has not been configured. do we want the event to occur when the filter is actually added, or just when clicking the `add filter` button

Reply:
> But the filter is not added until it has not been configured. 
True, but that component doesn't know if the filter is being added or edited.
The `onSave` property in FilterSearch is the one that adds the filter to the context. 
```
onSave={(value) => {
                        setSelectedField(undefined);
                        addDimensionDashboardFilter(value);
                        onClose();
                    }}
                    ```